spec/forms/schools/add_participant_form_spec.rb

```diff
@@ -57,7 +57,7 @@
     end
 
     context "when the email is not already in use" do
-      it "returns true" do
+      it "returns false" do
```

Comment:
this doesn't make much more sense to me (in relation to the body of the test), and I don't understand why the logic has reversed

Reply:
```
describe "email_already_taken?" do
  context "when the email is not already in use" do
    it "returns false"
      ...
    end
  end
end
```
makes a lot of sense to me. The method checking if an email is already taken should return false when the email is not already in use. 

The description was simply incorrect - the logic hasn't changed (and the test contents haven't either).